doc/ems-application-guide/src/energyplus-runtime-language/built-in-functions.tex

```diff
@@ -118,7 +118,7 @@ \subsection{Built-in Psychrometric Functions}\label{built-in-psychrometric-funct
 \midrule
 \endhead
 
-@RhoAirFnPbTdbW & Result & Density of moist air & kg/m < sup > 3 < /sup > \tabularnewline
+@RhoAirFnPbTdbW & Result & Density of moist air & \si{\kg\per\m\cubed} \tabularnewline
```

Comment:
@yujiex There are more places in the same table that need this fix (or similar). RhovFnTdbRh, RhovFnTdbWPb, VFnTdbWPb, and RhoH2O.

Reply:
My bad. Will fix them too.